src/Amplitude.php

```diff
@@ -179,39 +154,34 @@ public function resetQueue()
         return $this;
     }
 
-    /**
-     * Gets the event that will be used for the next event logged by call to logEvent() or queueEvent()
-     *
-     * You can also pass in an event or array of event properties.  If you pass in an event, it will be set as the
-     * event to be used for the next call to queueEvent() or logEvent()
-     *
-     * @param null|array|\Zumba\Amplitude\Event Can pass in an event to set as the next event to run, or array to set
-     *   properties on that event
-     * @return \Zumba\Amplitude\Event
-     */
-    public function event($event = null)
```

Comment:
This is pretty significantly changing how the library works, a large portion of the readme will also need to be rewritten to properly document how it works now.  I forget the reason the library held onto an event, maybe to mimic how the official JS library worked at the time or something.  Or maybe to make it easy for different parts of the code to add to an event before it is sent off.  Anyways, whatever the original reason was, if we do change that, the readme needs to be updated.

For instance, one section:

https://github.com/zumba/amplitude-php/blob/master/README.md#adding-user-properties-on-event-object

I'm OK with the change, as long as it does not remove functionality.  For instance, instead of getting a new event by calling `->event()`, you just create the event then pass it around, that is fine.  It is probably good to avoid single getter/setter methods (separation of concerns), if we made this library from scratch today we would probably do something similar.

But as I said the readme will need to be re-written to update the documentation.  Can't have a bunch of non-working example code. 😄